crates/protobuf_extractor/Cargo.toml

```diff
@@ -11,7 +11,7 @@ serde = { version = "1.0.200", features = ["derive"] }
 serde_json = "1.0.116"
 pretty_env_logger = "0.5.0"
 log = "0.4.21"
-tree-sitter = "0.19.3"
+tree-sitter = "0.22.5"
 tree-sitter-proto = { git = "https://github.com/mitchellh/tree-sitter-proto.git", rev = "42d82fa18f8afe59b5fc0b16c207ee4f84cb185f"}
```

Comment:
evidently this project hasn't updated in 3 years and locks us on 0.19 tree-sitter. :(